src/test/ui/on-unimplemented/enclosing-scope.rs

```diff
@@ -0,0 +1,27 @@
+// Tests scope annotions from `enclosing_scope` parameter
```

Comment:
Could you add the following line to the top of the file? This line will stop the compiler after it checks the code (before trying to codegen) and speed up CI slightly.

```
// check-pass
```

Reply:
I thought check pass is about `x.py check` being ok, is it not?

Reply:
`check-pass` is unrelated to `x.py check`, all it does is instruct the compiler to only do the equivalent of a `cargo check` for this test, since that’s enough for this test’s purposes.

Reply:
Yeah, sorry, `cargo check` is what I meant. My question was about the fact the test is expected to fail, should I use `check-pass` with `should-fail`, because as far as I understood `check-pass` tests are expected to pass.

Reply:
You’re totally right, it doesn’t need `check-pass`, nevermind, ignore this comment.